theme/templates/includes/res_type_col.html

```diff
@@ -1,67 +1,67 @@
 {% load hydroshare_tags %}
-
-<span class="resource-type-text">{{ resource|resource_type }}</span>
-{% if resource|resource_type == "Generic" %}
```

Comment:
what is the benefit to using . instead of | ?

Reply:
The benefit is that we are not making a function (template filter) call rather accessing a field/attribute of the resource